apps/src/templates/rubrics/EvidenceLevels.jsx

```diff
@@ -21,7 +21,7 @@ export default function EvidenceLevels({
     return (
       <div className={style.evidenceLevelSet}>
         <Heading6>{i18n.assignARubricScore()}</Heading6>
-        {evidenceLevels.map(evidenceLevel => (
+        {evidenceLevels.toReversed().map(evidenceLevel => (
```

Comment:
I'm a bit worried this solution is going to be too fragile. I actually thought I'd added an explicit ordering on the Rails side, but it looks like I didn't, so I think these are ordered server-side based on ID. Generally, this will be in order of understanding, so this would work, but I don't think that's guaranteed. Instead, I think it would make sense to explicitly sort here, instead of relying on an order from the server. The bonus is your tests should already cover that sorting :)

Reply:
Fair fair.  I updated it to be more independent from the way it is stored in the server.